c2corg_ui/static/js/search/pagination.js

```diff
@@ -133,6 +136,14 @@ app.PaginationController.prototype.goToLast = function() {
   var nextOffset = this.total - (this.total % this.limit);
   this.location_.updateFragmentParams({'offset': nextOffset});
   this.scope_.$root.$emit('searchFilterChange');
+  this.scrollToTop();
+};
+
+/**
+ * @private
+ */
+app.PaginationController.prototype.scrollToTop = function() {
```

Comment:
Since it is a private method, the naming convention is to have a trailing underscore: ``scrollToTop_``.